src/platform_impl/windows/event_loop.rs

```diff
@@ -213,8 +216,15 @@ impl<T: 'static> EventLoop<T> {
                 if 0 == winuser::GetMessageW(&mut msg, ptr::null_mut(), 0, 0) {
                     break 'main 0;
                 }
-                winuser::TranslateMessage(&msg);
-                winuser::DispatchMessageW(&msg);
+
+                let translated = self
+                    .translate_accel_callback
+                    .as_deref()
+                    .unwrap_or(&Box::new(|_| false))(msg);
```

Comment:
The fallback callback could be created once on creation of the eventloop avoiding having to reallocate it on each iteration.

Reply:
Great optimization, thanks. I think I can avoid allocating altogether for a slightly uglier code.